src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(colorSupplier.getRandomColor());
+        Random random = new Random();
+        ball.setNumber(random.nextInt(100));
```

Comment:
The `Random` object is correctly used without a seed, and `random.nextInt(100)` is appropriate for generating a random number. Ensure that this logic meets the task requirements for the range of random numbers.